migration/lpass.py

```diff
@@ -5,7 +5,7 @@
 
 def get_lp_data():
     # Sign user into LastPass
-    lp_username = input("Please enter your LastPass username ")
+    lp_username = input("Please enter your LastPass username.\nWe don’t store this and it will only be used during this session.\n")
```

Comment:
Based on https://github.com/1Password/solutions/pull/26, it seems that this line lost some tiny enhancements. I'm putting the suggestion here, but I'll leave it up to @volodymyrZotov and @scottisloud to decide if this is still a valid enhancement or not:
```suggestion
    lp_username = input("Please enter your LastPass username.\nWe don’t store this and it will only be used during this session.\n>")
```

Reply:
I actually think we need that enhancement and more. 
```suggestion
    lp_username = input("You will be prompted once for your LastPass username and twice for your Master Password.\nThe first prompt signs you into the LastPass CLI, the second prompt approves the export.\nWe don't store these credentials and will only be used for this session.\n\nPlease enter your LastPass username.\n>")
```

It's.... a lot, but I do think getting the Master Password prompt twice may be confusing so might be helpful to prepare the user for that. Not sure if there's a more elegant way of doing that though.

Reply:
Got a fancier idea then, We'd need a title and then a description, similar to what we do when installing a Shell Plugin.
Here's a sample screenshot with how it looks for Shell Plugins:

<img width="697" alt="image" src="https://user-images.githubusercontent.com/25607825/214270750-58b54cdd-f6e6-49bd-81e5-88c0c015a319.png">

We can do a similar thing here as well. Not sure how easy it can be done in Python and it's more important to send the message now rather than its format, but it can be done in a separate PR to add bolds and coloring to this migration. It makes a difference in terms of visuals and where the users' focus should go.

Reply:
I tried different scenarios but never asked to enter master password twice despite I was logged in in `lpass` or not.
Interesting why you asked for it 2 times...

Reply:
> I tried different scenarios but never asked to enter master password twice despite I was logged in in `lpass` or not. Interesting why you asked for it 2 times...

I think it depends on the current state of the lastpass CLI session. Our code first does `lpass login`, then `lpass export`. Both commands solicit a Master Password, but I suspect if you have an existing session (so if you `lpass login` sometime in the recent past) then that may not generate a prompt, but the `lpass export` would, so you would only see one prompt. At least that's kind of been my experience. Not the biggest of deals, per se, so I think we can just use feedback to help set the user's expectation.

Reply:
@edif2008 let's consider that for a future update. If we can swing something like that, awesome, but I think we'd be wise to get some of the existing improvements in this MR out the door.

Reply:
Indeed, that is for a future update. I just shared it to see where this could go as a possible solution, but definitely worth getting these changes out the door. How we show the messages can be done in a separate PR.